### ISSUE TITLE
Help to toggle WASD to arrow keys. Only while Caps lock active

### ISSUE THREAD

**Issue by epecho** [pqrs-org/KE-complex_modifications#297]
Hey guys, Love this program and thankful for any help

Like the title states, I have my left hand on my SHIFT KEY, OPTION KEY and COMMAND key all the time. but I hate when I have to use my arrow keys to move shapes. I would like the ability to change WASD as arrow keys to nudge shapes around. This also allows for the pinky to hit the left SHIFT KEY so that I could move shape 10px at a time. However, I want to be able to toggle to the arrow function by pressing Caps lock and holding it until I release. Doing this will enable me to make more use of hotkeys while building/UI.

To summarize:
1. Program active
2. keyboard is in default state when Caps lock is off
3. While Caps lock is on I am able to use WASD as arrow keys
4. Once I hit Caps lock, keys will return to default state.

Thank you so much for whatever help you guys can throw my way. Hope this has already been addressed and is easy fix
